scripts/pam_swap.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to handle swapping of PAMs.
+"""
```

Comment:
It's not just about pulling out generic code, even specific code can be in a module that can be tested. The problem is if you have a lot of logic in scripts, there's no good way to actually test the logical branches.